sources/install/package_ad.sh

```diff
@@ -63,6 +63,15 @@ function install_sprayhound() {
     add-to-list "sprayhound,https://github.com/Hackndo/Sprayhound,Active Directory password audit tool."
 }
 
+function install_asrepcatcher() {
+    # CODE-CHECK-WHITELIST=add-aliases
+    colorecho "Installing ASRepCatcher"
+    pipx install git+https://github.com/Yaxxine7/ASRepCatcher
```

Comment:
Hello, can you add --system-site-packages flag ?

Reply:
Hello, it's done. Can you tell me why was it necessary ?

Reply:
It's a pretty new requirement
In order to avoid having all python virtual environments duplicate the same dependencies over and over again we installed the most common ones system-wide and now create the venvs with that in context

Reply:
Sorry @Yaxxine7 but can you add this flag before url ? -> pipx install --system….. https://….

Reply:
Ok, no problem, it's done.